Sources/MongoSwift/MongoCollection.swift

```diff
@@ -84,11 +95,17 @@ public class MongoCollection<T: Codable> {
         self._collection = nil
     }
 
-    /// Drops this collection from its parent database.
-    /// - Throws:
-    ///   - `ServerError.commandError` if an error occurs that prevents the command from executing.
-    public func drop(session: ClientSession? = nil) throws {
-        let operation = DropCollectionOperation(collection: self, session: session)
+   /**
+    *   Drops this collection from its parent database.
+    * - Parameters:
+    *   - session: An optional `ClientSession` to use when executing this command
+    *   - options:  Optional `DropCollectionOptions` to use when executing this command
```

Comment:
[nit] This line has an extra space preceding "Optional" that can be removed. Also, the first part can be slightly rewritten to correspond with the "session" description ("An optional `DropCollectionOptions` ... <rest the same as it is>")